controls/views.py

```diff
@@ -5,7 +5,8 @@
 from django.conf import settings
 from django.contrib import messages
 from django.contrib.messages.views import SuccessMessageMixin
-from django.shortcuts import render
+from django.shortcuts import get_object_or_404, redirect, render
+from django.contrib.auth.decorators import login_required
```

Comment:
This is a duplicate of line 13 below, can be removed. (This was recently merged into develop.)

Reply:
Agreed. I suggested changes to optimize imports.